Gruntfile.js

```diff
@@ -408,19 +319,59 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-github-releaser');
   grunt.loadNpmTasks('grunt-aws-s3');
   grunt.loadNpmTasks('grunt-contrib-uglify');
+  grunt.loadNpmTasks('grunt-browserify');
+  grunt.loadNpmTasks('grunt-coveralls');
+  grunt.loadNpmTasks('grunt-contrib-copy');
+  grunt.loadNpmTasks('grunt-contrib-concat');
+  grunt.loadNpmTasks('grunt-exorcise');
+
+  grunt.registerTask('build', [
+    'clean:build',
+    'jshint',
+    'browserify',
+    'copy:novtt',
+    'concat:vtt',
+    'exorcise',
+    'uglify',
+    'less',
+    'version:css',
+    'cssmin',
+    'copy:fonts',
+    'copy:swf',
+    'vjslanguages'
+  ]);
+
+  grunt.registerTask('dist', [
+    'clean:dist',
+    'build',
+    'copy:dist',
+    'copy:examples',
+    'zip:dist'
+  ]);
 
-  // grunt.loadTasks('./docs/tasks/');
-  // grunt.loadTasks('../videojs-doc-generator/tasks/');
+  grunt.registerTask('cdn', [
+    'dist',
+    'copy:cdn',
+    'dist-cdn'
+  ]);
+
+  // Remove this and add to the test task once mmcc's coverall changes are merged
+  grunt.registerTask('newtest', ['build', 'karma:chrome']);
 
-  grunt.registerTask('pretask', ['jshint', 'less', 'vjslanguages', 'build', 'usebanner', 'uglify']);
   // Default task.
-  grunt.registerTask('default', ['pretask', 'dist']);
-  // Development watch task
-  grunt.registerTask('dev', ['jshint', 'less', 'vjslanguages', 'build', 'usebanner', 'qunit:source']);
-  grunt.registerTask('test-qunit', ['pretask', 'qunit']);
+  grunt.registerTask('default', ['build', 'test']);
+
+  // Development watch task. Doing the minimum required.
+  grunt.registerTask('dev', ['jshint', 'less', 'browserify', 'karma:chrome']);
 
-  grunt.registerTask('dist', 'Creating distribution', ['dist-copy', 'zip:dist']);
+  // Tests.
+  // We want to run things a little differently if it's coming from Travis vs local
+  if (process.env.TRAVIS) {
+    grunt.registerTask('test', ['build', 'test-travis', 'coveralls']);
```

Comment:
Yes, every test run shows the coverage in the test output, and there's also a txt file break down generated in `test/coverage/coverage.txt`. Generating the HTML file unfortunately throws errors right now and I don't think it's worth the time to debug.